utils/deprecations.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from os.path import basename
+
+log = logging.getLogger(__name__)
```

Comment:
Where is this logger outputting messages? I don't know is there is a `utils.` handler setup or a catchall, did you test it?